index.js

```diff
@@ -1,6 +1,15 @@
-var ExceptionLogger = require('./lib/exception-logger');
+var ExceptionLogger = require('./lib/exception-logger'),
+    conman = require('./lib/con-man');
 
 
 module.exports = function(api_token) {

```

Comment:
yes. it exports a class constructor, which is traditionally `LikeThis` in node, and (i think) the entirety of `js`